docs/pages/cloud/faq.mdx

```diff
@@ -55,6 +55,16 @@ $ tsh login --proxy=myinstance.teleport.sh:443
 $ tctl status
 ```
 You must use the enterprise version of `tctl`. 
+
+### Why am I getting `permission denied` errors when using `tctl`?
+
+If you have a local file `/etc/teleport.yaml` on your machine `tctl` will attempt to use the local cluster. Set the environment variable `TELEPORT_CONFIG_FILE` to `""` so it will not attempt to use that Teleport configuration file.
+
+```bash
+$ export TELEPORT_CONFIG_FILE=""
```

Comment:
I think we'll be appending the `$` and other code blocks programmatically in the future. There's an edit pass in place to get the code snippets that're already there. I can edit the last of these in one final pass in the future.